lib/controller/page/type/page.summary/page.summary.controller.js

```diff
@@ -160,15 +164,18 @@ SummaryController.attachEmailSubmissions = async (submissionId, userData, submis
   }
 }
 
-SummaryController.attachJsonSubmission = async (submissions) => {
+SummaryController.attachJsonSubmission = async (submissions, userData) => {
   if (SERVICE_OUTPUT_JSON_ENDPOINT === undefined) {
     return
   }
 
+  const callbackUrl = `http://${SERVICE_SLUG}.formbuilder-services-${PLATFORM_ENV}-${DEPLOYMENT_ENV}:${PORT}/api/submitter/json/default/${userData.getUserId()}.json`
+
   const submission = {
     type: 'json',
     url: SERVICE_OUTPUT_JSON_ENDPOINT,
-    encryption_key: SERVICE_OUTPUT_JSON_KEY
+    encryption_key: SERVICE_OUTPUT_JSON_KEY,
+    callback_url: callbackUrl
```

Comment:
should these be snake or camel cased?

Reply:
camelCased for the js, snake_cased for the ruby-backed endpoint